packages/visual-editor/src/hooks/useComponentProps.ts

```diff
@@ -357,3 +375,51 @@ const addExtraDropzonePadding = (padding: string) =>
 
 const isPercentValue = (value?: string | number) =>
   typeof value === 'string' && value.endsWith('%');
+
+const calculateDesignVariableValue = ({
```

Comment:
With no code comment, it might not be easy for other devs to judge what this function is actually doing. Can you think of a name that somehow represents its logic of looking up the default value from its wrapping pattern definition? Example `maybeMergePatternDefaultDesign`? (I know, very clumsy name ^^)